src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStore.java

```diff
@@ -283,6 +296,21 @@ public URL toExternalURL(@NonNull Blob blob, @NonNull HttpMethod httpMethod) thr
     @Override
     public Map<String, URL> artifactUrls(Map<String, String> artifacts, Map<String, String> contentTypes, BlobStore blobStore, String key) throws IOException {
         Map<String, URL> artifactUrls = new HashMap<>();
+        
+        if (!getConfiguration().getGeneratePresignedUrls()) {
```

Comment:
Using simple public URLs for _download_ is one thing but how could this possibly work for https://github.com/jenkinsci/artifact-manager-s3-plugin/blob/2567c4c52d797921eefbf4298deaa3137b1b377a/src/main/java/io/jenkins/plugins/artifact_manager_jclouds/JCloudsArtifactManager.java#L127-L128?